src/Components/Server/src/DependencyInjection/ComponentServiceCollectionExtensions.cs

```diff
@@ -47,6 +49,9 @@ public static IServerSideBlazorBuilder AddServerSideBlazor(this IServiceCollecti
                 options.SupportedProtocols.Add(BlazorPackHubProtocol.ProtocolName);
             });
 
+            // Matcher policy to configure csrf protection on the negotiate endpoint
```

Comment:
It's not really CSRF, is it? My understanding was that we're protecting against circuit ID exfiltration via XSS.

Reply:
We're protecting against the circuit id being used outside of the browser context, if you manage an XSS and get access to the request token you have everything you need to connect to the server and do stuff as you are effectively running in the users context. (Although you need to do it from within the page, which I'm not sure is viable). 

The cookie prevents against the circuit id being used from outside the browser context and across origins (as is same-site by default, although this is something likely we will allow configuring, as its required based on conversations I had with Chris)

Reply:
I understand - my comment was just about the use of the term `csrf` in the line I'm commenting on. It seems unrelated to what we're doing here.